plugins/org.jboss.tools.locus.dmr/1.3.0.Final/src/org/jboss/dmr/Activator.java

```diff
@@ -0,0 +1,19 @@
+/*************************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc. and others.
+ * All rights reserved. This program and the accompanying materials 
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ *     JBoss by Red Hat - Initial implementation.
+ ************************************************************************************/
+package org.jboss.dmr;
+
+/**
+ * This class is a placeholder to avoid compilation errors in this project
+ * after importing into eclipse workspace.
+ */
+public class Activator {
```

Comment:
This is a pretty simply change to the code, but as Mickael said, we shouldn't really have to ADD code to the Locus bundles. Need a +1 from @maxandersen or @alexeykazakov here if we're going to set precedent and fork from what's upstream in the maven bundle. 

Could this change be done upstream in dmr 1.3.x or 1.4.x, instead?